src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,50 @@
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  transform: translate(40vw, 30vh);
+  height: $clock-diametr;
+  width: $clock-diametr;
```

Comment:
You watch is circle, if you gonna to use it on other pages, better to use `@mixin `